cpp/include/cudf/detail/copy.hpp

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2018-2021, NVIDIA CORPORATION.
+ * Copyright (c) 2019-2022, NVIDIA CORPORATION.
```

Comment:
This was an oversight. Another vim snafu. I'll fix this shortly.
(Yes, yes. I know, Emacs.)